src/bp-templates/bp-nouveau/buddypress/common/js-templates/activity/parts/bp-activity-post-case-heading.php

```diff
@@ -0,0 +1,7 @@
+<script type="text/html" id="tmpl-activity-post-case-heading">
+	<# if ( data.display_avatar ) {  #>
+		<span class="activity-post-user-name-container">
+            <h5><a class="activity-post-user-name" href="{{data.user_domain}}"><span class="user-name">{{data.user_display_name}}</span></a></h5>
+        </span>
```

Comment:
@annrra @amincdev same here make the code proper alignment.

Reply:
Thanks @chetansatasiya 
Done - "No fixable errors found"